ODC/datapackage.json

```diff
@@ -0,0 +1,20 @@
+{
+	"name": "COVID-19-testing-locations-ODC",
+	"datapackage_version": "0.1.0",
```

Comment:
The datapackage_version field (which doesn't actually seem to be documented) is used to track the version of the datapackage schema itself that this file is governed by. It seems that there has only ever been one version of this spec so if we list it it should be 1.0.0
 The actual data itself has a version described by a standard "version" field

Reply:
I see. I removed "datapackage_version" field so we don't need to keep track of the version change on the datapackage schema.

The "version" field is added.